### ISSUE TITLE
Unable to deserialize response when getting user log events with includeTotals: false

### ISSUE THREAD

**Issue by mattstewart** [auth0/auth0.net#184]
The code:

```
var managementApiClient = new ManagementApiClient(TOKEN, DOMAIN);
await managementApiClient.Users.GetLogsAsync(USERID, page: 0, perPage: 1, sort: "date:-1", includeTotals: false);
```
Returns no log entries for the user. During deserialization, the PagedListConverter class is looking for a "_collectionFieldName" of "logs", which does not exist in the API response. 

Upon further investigation it seems that the entire structure of the data returned from APIv2 when `include_totals=false` is passed is completely different. For example...

```
GET /users/{user_id}/logs?include_totals=true
```
...returns a logs array of each event...

```
{
  "length": 4,
  "total": 4,
  "start": 0,
  "limit": 5,
  "logs": [
    {
      "date": "2018-04-19T23:31:33.947Z",
      "type": "s",
      ...
    },
    {
      "date": "2018-04-19T23:31:26.907Z",
      "type": "scp",
      ...
    },
  ...
}
```

However...

```
GET /users/{user_id}/logs?include_totals=false
```
...returns each log event as a separate field on the returned object…
```
{
  "0": {
    "date": "2018-04-19T23:31:33.947Z",
    "type": "s",
    ...
  },
  "1": {
    "date": "2018-04-19T23:31:26.907Z",
    "type": "scp",
    ...
  },
  ...
}
```

**Comment by jerriep** [comment]
@mattstewart Is there actually data being returned by that API call? 

Your C# code sample sends extra parameters which you do not appear to be sending when you call the endpoint directly. 

To try and track this down, please inspect the request and response of the API call made by your C# application in something like Fiddler and see whether the actual payload returned from Auth0 contains entries.

**Comment by jerriep** [comment]
@mattstewart Please ignore my last email...

As you correctly noted, the structure is very different from what the C# SDK expects. It expects an array, but it gets an object structure. I will need time to investigate how to deserialize that correctly, so I may not have a fix for you very quickly.

For now, to work around this and get the log entries for a user, you can use the normal logs endpoint, and pass the user_id in the query. This is what the C# code will look like:

```
var logEntries = await managementApiClient.Logs.GetAllAsync(q: "user_id:\"USER_ID\"");
// e.g.
var logEntries = await managementApiClient.Logs.GetAllAsync(q: "user_id:\"auth0|92004....\"");
```

**Comment by jerriep** [comment]
Or if you need to use `Users.GetLogsAsync` then set `includeTotals` to true (until I can fix this...)

```
await managementApiClient.Users.GetLogsAsync(USERID, page: 0, perPage: 1, sort: "date:-1", includeTotals: true);
```

**Comment by jerriep** [comment]
@mattstewart Please grab version 5.3.0 which solves this bug

**Comment by mattstewart** [comment]
@jerriep Thank you for the quick turnaround! I can confirm that the issue has been addressed in v5.3.0.
